text/processing.py

```diff
@@ -280,16 +280,16 @@ class WriteToTextFileJob(Job):
     Write a given content into a text file, one entry per line
     """
 
-    def __init__(
-        self,
-        content,
-    ):
+    __sis_hash_exclude = {"out_name": "file.txt"}
```

Comment:
```suggestion
    __sis_hash_exclude__ = {"out_name": "file.txt"}
```
to fix the broken hashes in the pipeline.

Reply:
oopsi ...